public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Question.js

```diff
@@ -29,6 +29,12 @@ export class StoryQuestionsQuestion extends React.Component {
       this.props.fieldValue.answers : []
   }
 
+  searchSuggestionsId = this.genKey()
+
+  genKey() {
+    return 'xxxxxxx'.replace(/x/g, () => (Math.random() * 36 | 0).toString(36));
```

Comment:
What's the value of `| 0`?

Reply:
Truncation of the decimal part.

Reply:
Thanks. I wonder whether the whole method can be defined as `Math.random().toString(36)`. It seems to have a similar result in both Chrome and Firefox.

Reply:
Definitely 👍